packages/dds/tree/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -293,8 +308,7 @@ class RebaseQueue<T> {
 					}
 				}
 			}
-			const revision = baseMark.revision ?? this.baseMarks.revision;
-			const reattachOffset = getOffsetAtRevision(newMark.lineage, revision);
+			const reattachOffset = getOffsetAtRevision(newMark.lineage, this.baseIntention);
```

Comment:
We will have to change this if we ever support rebasing over compositions, right?

Reply:
Correct. There's a lot of code that would have to change, I think.